### ISSUE TITLE
With regards to the tutorials link...

### ISSUE THREAD

**Issue by huhwhatthe** [love2d-community/awesome-love2d#212]
The first one leads to some weird love letter for Hitler...? WTF is this, ironically linked from 'love'?

**Comment by Zireael07** [comment]
I checked three first links in the tutorial section (in private browsing mode) and I'm seeing nothing of the sort. Please post the link you're referring to and double check you don't have some malware on your computer redirecting things.

**Comment by refeals** [comment]
I'm not sure I can find it either. The links are fine.

**Comment by huhwhatthe** [comment]
What malware, if you're willing to provide a huge list of links then the least you could do is know what they have... it's https://github.com/a327ex/blog/issues/70 which is the new username of that 'adn' you got listed... first in list, and for someone who's just trying to learn all about this software... HUGE turn off. If this adn/a327 has some boner for Hitler then no one should know about it, much less on some supposed programming tutorials!

**Comment by Zireael07** [comment]
Ah, I didn't check every one of the guy's blogposts. All the others are completely programming-focused.

**Comment by slime73** [comment]
That guy is banned from most of love's communities for similar reasons as that. Thanks for bringing that to my attention.

**Comment by Grinshpon** [comment]
I just read the blog post you mentioned and I see no mention of Hitler nor any kind of love shown towards him or his regime. So this entire issue seems to be predicated on misinformation which is a poor look

**Comment by slime73** [comment]
> So this entire issue seems to be predicated on misinformation which is a poor look

Not at all, the person in question has been using LÖVE for over a decade and has a very extensive track record of this kind of thing, although not all of it has survived across that long duration. There has been no remorse or willingness to change for the better at any point, unfortunately.

**Comment by Sekitan** [comment]
I have to agree with @Grinshpon I don't see any such "love for Hitler" in the linked blog post and the only person making the claim that such content is in there is @huhwhatthe 

The only time Hitler is "referenced" is when the blog post praises Steam for its willingness to host potentially controversial content since they allow the ridiculously named "Hentai Nazi" game on their platform.
The author even explicitly states it's not about the game being about Nazis:

> It's a good example not because it's about hentai or about nazis, but because these are both contentious subjects

If there was some previous edit of the blog post where the author said the opposite I would understand but I don't see any evidence of that.  Further, how does this blog post justify the removal of a number of unrelated tutorials created by the same author?  Do these tutorials mention Hitler as well?  If not then what is wrong with the tutorials that would require their removal?

@slime73 mentioned just now that the author "has a very extensive track record of this kind of thing".  "This kind of thing" meaning what?  NOT proclaiming a love for Hitler?  Making blog posts about a controversial subject?  There's apparently an "extensive track record" so examples of these issues should be easy to find, and if so, should have been directly linked to this issue if we're going to justify purging all of the author's tutorials.

Finally, assuming the tutorials were at least of decent quality, the only people being punished here are the Love2D users who are now deprived of possibly useful tutorials because one guy made a false claim about the content of an unrelated blog post and another guy chimed in with "yeah he totally does it all the time".

I'm seriously struggling to understand the justification here.

**Comment by slime73** [comment]
> @slime73 mentioned just now that the author "has a very extensive track record of this kind of thing". "This kind of thing" meaning what? 

Bigotry and hate towards marginalized groups. I have no interest in making love's communities less safe for marginalized folks, and I have no interest in promoting his views. 

> There's apparently an "extensive track record" so examples of these issues should be easy to find, and if so, should have been directly linked to this issue if we're going to justify purging all of the author's tutorials.

No, that's not how community moderation works. It's not a group vote. Like I said he's been around for much longer than his blog, and he's been banned from various places for much longer as well. If you want to you can find one of his other social media accounts to see him stating opinions fairly recently such as: white people have inherently higher IQ than black people.

That's just one "small" example. I hope you're well versed enough in those talking points to understand how much of a dog whistle it is. Now multiply that by dozens of other bigger examples.

> the only people being punished here are the Love2D users

If people like him were promoted or allowed to participate in love's communities, people who his views target would be punished by being less safe in those communities. love would be punished by being known as unsafe. It's too bad he's a bad person, I wish he would change. He has not.

> I'm seriously struggling to understand the justification here.

It's good that someone dropping by who lacks context doesn't need to understand the justification for it to be justified. :)

**Comment by a327ex** [comment]
@Sekitan @Grinshpon You guys shouldn't argue, if @slime73 doesn't want me around it's fine. It's very understandable that my blog posts can be incendiary when read by some people, although I still don't understand what exactly I did in any LÖVE community that warrants the description "very extensive track record of bigotry and hate towards marginalized groups", and slime didn't explain to me either when I was banned years ago. Still, it's not my community, and if I'm not welcome I'm not going to argue.

>white people have inherently higher IQ than black people.
I don't remember where I said this, but if I did say it it wasn't in any LÖVE community, as I don't really talk about these subjects in places that aren't fit for it. If you're stalking me and reading my comments on random places online and then assuming I will behave similarly in any community I'm in then that's just a mistake, but it's your mistake to make.
